test/jasmine/tests/plots_test.js

```diff
@@ -494,4 +494,49 @@ describe('Test Plots', function() {
             assert(dest, src, expected);
         });
     });
+
+    describe('Plots.graphJson', function() {
+
+        it('should serialize data, layout and frames', function(done) {
+            var mock = {
+                data: [{
+                    x: [1, 2, 3],
+                    y: [2, 1, 2]
+                }],
+                layout: {
+                    title: 'base'
+                },
+                frames: [{
+                    data: [{
+                        y: [1, 2, 1],
+                    }],
+                    layout: {
+                        title: 'frame A'
+                    },
+                    name: 'A'
+                }, {
```

Comment:
Perhaps add a `null` or otherwise garbagy frame just to confirm/more or less lock in the behavior there?